index.html

```diff
@@ -223,36 +223,40 @@ <h3>Skills</h3>
       </main>
       <nav class="app-grid__fixed-btn nav">
         <ul class="nav--display-content">
-          <li data-js="home-button" class="no-bul nav__icon">
+          <li class="no-bul nav__icon">
```

Comment:
Just an idea, but you could use buttons here with an image inside instead of an unordered list :)